var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -28,14 +28,21 @@ class Lammps(CMakePackage, CudaPackage):
     #   marked deprecated=True
     # * patch releases older than a stable release should be marked deprecated=True
     version("develop", branch="develop")
+    version("20230208", sha256="60221242145da4479e5b207d9a0eed90af4168d7a297b4dc8c0e7f2b3215602e")
     version("20221222", sha256="75372ee7ef982767fc4ed4dc95e20ddca8247419adeb0c1276c40e43d1eab955")
     version("20221103", sha256="d28517b84b157d4e46a1a64ed787b4662d8f2f5ade3f5a04bb0caed068f32f7e")
     version("20220915", sha256="392b8d35fc7919b0efaf8b389259a9b795a817e58e75e380467c63d03a0dab75")
     version("20220803", sha256="f37cb0b35c1682ffceae5826aadce47278aa7003099a1655fcea43acd7d37926")
+    version(
+        "20220623.3",
+        sha256="8a276a01b50d37eecfe6eb36f420f354cde51936d20aca7944dea60d3c098c89",
```

Comment:
`spack checksum lammps@20220623.3` for me is: `d41cd5a4dfdd68e98eba4e947da8ed159c623a2862e1f75e4d262b36ae74492e`.

Reply:
weird: what file is it downloading? here is my output:
```
$ spack checksum lammps@20220623.3
==> Found 1 version of lammps:
  
  20220623.3  https://github.com/lammps/lammps/archive/stable_23Jun2022_update3.tar.gz

==> Fetching https://github.com/lammps/lammps/archive/stable_23Jun2022_update3.tar.gz

    version("20220623.3", sha256="8a276a01b50d37eecfe6eb36f420f354cde51936d20aca7944dea60d3c098c89")


```

Reply:
Note, in order for it to work the version must be listed in the package.py and have `preferred` set to `True`. Otherwise, it will default to downloading a `patch_` tarball.

Reply:
That's what it seems to be downloading:

```https://github.com/lammps/lammps/archive/patch_23Jun2022_update3.tar.gz
```

Reply:
Nvm.  Able to confirm now.